stage1-coreos panics when image has empty exec

My initial experience with `rkt` was a little jarring:

```console
$ sudo rkt run quay.io/listhub/alpine                      
rkt: using image from local store for image name coreos.com/rkt/stage1-coreos:0.13.0
rkt: using image from local store for image name quay.io/listhub/alpine
panic: runtime error: index out of range

goroutine 1 [running, locked to thread]:
main.(*Pod).appToSystemd(0xc8200188a0, 0xc820088fc0, 0x0, 0xc82011aa10, 0x6, 0x0, 0x0, 0x0, 0x0)
        /home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/pod.go:308 +0x5fab
main.(*Pod).PodToSystemd(0xc8200188a0, 0x0, 0xc82011aa10, 0x6, 0x0, 0x0, 0x0, 0x0)
        /home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/pod.go:465 +0xf9
main.stage1(0x7fcd556703a0)
        /home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/init.go:617 +0x18e5
main.main()
        /home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/init.go:908 +0x42

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1696 +0x1
```

I eventually learned that, if I was expecting a shell, I'd have needed to

 1. use `--interactive` 
 2. either use `--exec` or `acbuild` myself an image with a non-empty `exec`

but my first thought was that something was wrong with my `rkt` installation. :grin:

(rkt version 0.13.0, appc version 0.7.3)

--- a/stage1/init/pod.go
+++ b/stage1/init/pod.go
@@ -249,6 +249,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool, flavor strin
 	}
 	imgName := image.Name
 
+	if len(app.Exec) == 0 {
+		return fmt.Errorf(`image %q has an empty "exec" (try --exec=BINARY)`, imgName)
+	}
+
 	workDir := "/"
 	if app.WorkingDirectory != "" {
 		workDir = app.WorkingDirectory

